test/core/crawlers/cheerio_crawler.test.ts

```diff
@@ -1151,43 +1150,6 @@ describe('CheerioCrawler', () => {
             expect(warningSpy).toBeCalledWith(`Found cookies with similar name during cookie merging: 'Foo' and 'foo'`);
             expect(warningSpy).toBeCalledWith(`Found cookies with similar name during cookie merging: 'coo' and 'Coo'`);
         });
-
-        test('should use sessionId in proxyUrl when the session pool is enabled', async () => {
```

Comment:
Why is this expectation no longer valid? For Apify proxy, the session ID is pretty important. Will you move the functionality to the SDK?